game/scripts/npc/items/blinks/item_arcane_blink.txt

```diff
@@ -72,7 +72,7 @@
       "duration"    "6"
       "cast_pct_improvement"    "35 40 45 50 55" //OAA
       "manacost_reduction"    "25 30 35 40 45"
-      "debuff_amp"          "20 23 26 29 32"
```

Comment:
y not tho

Reply:
scaling on scaling is stupid. if arcane blink is weak it can be changed other ways

Reply:
id remove manacost or stop that scaling. its the more ueselss option